test/e2e/cypress.ab_runtime.json

```diff
@@ -0,0 +1,15 @@
+{
+  "baseUrl": "http://localhost:8088",
+  "defaultCommandTimeout": 12000,
+  "env": { "stack": "test_ab" },
+  "ignoreTestFiles": [
+    "**/*app/*",
+    "**/0004_*.js",
```

Comment:
I'm not sure about ignoring by number. Could we put the designer test in a folder to ignore?